src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/aemformscontainer/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This proxy component should be deleted when the forms flag is not set.

Reply:
thanks for the review @bpauli. I have added this logic.